plf-packaging-resources/src/main/resources/CHANGE_LOG.txt

```diff
@@ -6,136 +6,113 @@ Changelog Platform ${org.exoplatform.platform.version}
+           
+eXo GateIn ${org.exoplatform.platform.version}
 *************************
 
 ** Bugs
-    * CAL-1249 - [Security] calendar import got XSS issue
-    * CAL-1267 - [Calendar Integration] Performance regression for calendar read home
-    * CAL-1272 - URLS not clickable in events popover
-    * CAL-1273 - Unknown error in Advanced search
+    * EXOGTN-2267 - Problem of capital letter
+    
+** Improvements
+    * EXOGTN-2282 - Improve syncAll reset service logs	
 
-eXo Integration ${org.exoplatform.platform.version}
+eXo Calendar ${org.exoplatform.platform.version}
 *************************
 
 ** Bugs
-    * INTEG-461 - An offline user can disable the search UnifiedSearch
+    * CAL-1219 - Reminder with "Repeat Every" does not work correctly 
+    * CAL-1282 - cannot import remote calendar
+    * CAL-1283 - [Regression] Problem with single quote encode in title event
+	
+** Improvements
+    * CAL-1281 - Calendar checkbox status is not saved after refreshing the page 
+
 
 eXo Forum ${org.exoplatform.platform.version}
 *************************
 
 ** Bugs
-    * FORUM-1084 - The test case org.exoplatform.forum.service.TopicTestCase.testTopic UNSTABLE
-    * FORUM-1318 - Topic edition takes wrong person in activity comment
-    * FORUM-1319 - [Private Messages] Problem in accessing Private Messages when delete user
+    * FORUM-1347 - Error when trying to download an attachment twice
+	
+eXo Core ${org.exoplatform.platform.version}
+*************************
 
-eXo GateIn ${org.exoplatform.platform.version}
+**Bugs
+    * COR-361 - Possibiliy to share Conversation State between two users
+	
+===============================
+Changelog Supported Addons
+===============================
+
+
+
+eXo Add-on CHAT 1.5.1
 *************************
 
-** Improvements
-    * EXOGTN-2149 - Add validator for property of type Double
-    * EXOGTN-2249 - PL IDM DB only mode should be automatically detected
+**Bugs
+     * CHAT-742 - Chat not working if there is a space after the value of the property standaloneChatServer  
+     * CHAT-749 - Improve Chat server logs in ServerBootstrap
+     * CHAT-755 - User chat status is not updated after session timeout
+	 
+eXo Add-on Task 1.2.5
+*************************
 
-** Bugs
-    * EXOGTN-2169 - [Unified Search] Search Page doesn't keep the query after language switch
-    * EXOGTN-2223 - JBID_IO_ATTR_TEXT_VALUES.TEXT_ATTR_VALUE_ID column should be indexed
-    * EXOGTN-2234 - Template statistics collection use blocker operation
-    * EXOGTN-2236 - Remove unnecessary bottleneck from ServletContainer
-    * EXOGTN-2241 - server is blocked while starting
-    * EXOGTN-2248 - Fix DescriptionService cache to cache null values
\ No newline at end of file
+**Bugs
+     * TA-571 - Filter does not work for the first time
+     * TA-574 - [Tasks] Warning message when trying to modify task's status
+	 
+** Improvements
+     * TA-213 - The Group by & Sort by view are changed after adding tasks
+	 
```

Comment:
The changelog is added in community and entreprise editions, which do not contain chat and task, so I would not add them.